email.rst

```diff
@@ -4,102 +4,57 @@
 
-        <!-- app/config/config.xml -->
-        <?xml version="1.0" encoding="UTF-8" ?>
-        <container xmlns="http://symfony.com/schema/dic/services"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns:swiftmailer="http://symfony.com/schema/dic/swiftmailer"
-            xsi:schemaLocation="http://symfony.com/schema/dic/services
-                http://symfony.com/schema/dic/services/services-1.0.xsd
-                http://symfony.com/schema/dic/swiftmailer http://symfony.com/schema/dic/swiftmailer/swiftmailer-1.0.xsd">
-
-            <swiftmailer:config
-                transport="%mailer_transport%"
-                host="%mailer_host%"
-                username="%mailer_user%"
-                password="%mailer_password%"
-            />
-        </container>
-
-    .. code-block:: php
+.. _swift-mailer-configuration:
 
-        // app/config/config.php
-        $container->loadFromExtension('swiftmailer', array(
-            'transport'  => "%mailer_transport%",
-            'host'       => "%mailer_host%",
-            'username'   => "%mailer_user%",
-            'password'   => "%mailer_password%",
-        ));
+Configuration
+-------------
 
-These values (e.g. ``%mailer_transport%``), are reading from the parameters
-that are set in the :ref:`parameters.yml <config-parameters.yml>` file. You
-can modify the values in that file, or set the values directly here.
+The ``config/packages/swiftmailer.yaml`` file created when installing the mailer
+provides all the initial config needed to make it work, except the parameters
+required to connect to the mail server. Those parameters are defined in the
+``MAILER_URL`` environment variable in the ``.env`` file:
 
-The following configuration attributes are available:
+.. code-block:: bash
 
-* ``transport`` (``smtp``, ``mail``, ``sendmail``, or ``gmail``)
-* ``username``
-* ``password``
-* ``host``
-* ``port``
-* ``encryption`` (``tls``, or ``ssl``)
-* ``auth_mode`` (``plain``, ``login``, or ``cram-md5``)
-* ``spool``
+    # use this to disable the email delivery
+    MAILER_URL=null://localhost
 
-  * ``type`` (how to queue the messages, ``file`` or ``memory`` is supported, see :doc:`/email/spool`)
-  * ``path`` (where to store the messages)
-* ``delivery_addresses`` (an array of email addresses where to send ALL emails)
-* ``disable_delivery`` (set to true to disable delivery completely)
+    # use this to send emails via Gmail (don't use this in production)
+    MAILER_URL=gmail://username:password@localhost
 
-.. caution::
+    # use this to configure a traditional SMTP server
+    MAILER_URL=smtp://localhost:25?encryption=ssl&auth_mode=login&username=&password=
 
-    Starting from SwiftMailer 5.4.5, the ``mail`` transport is deprecated
-    and will be removed in version 6. Consider using another transport like
-    ``smtp``, ``sendmail`` or ``gmail``.
+Refer to the :doc:`SwiftMailer configuration reference </reference/configuration/swiftmailer>`
```

Comment:
Is there an easy example we could show for Mandrill, SendGrid or Amazon SES? I believe you basically use `smtp`, but point to their servers. But that might not obvious without an example.

And if we show that here, could we delete `cloud.rst`?

Reply:
Yes, let's do that when we remove the other articles.